File: responses/CAXZU5-45T/sum_of_certain_multiples.js
function solution(x) {
  if (!Number.isInteger(Number(x))) {
    return null;
  }

  const values = [];
  let sum = 0;

  let subject = x - 1;
  while (subject > 0) {
    if (subject % 3 && subject % 5) {
      subject = subject - 1;
      continue;
    } else if (subject % 3 || subject % 5) {
      sum += subject;
      values.push(subject);
    }

    subject = subject - 1;
  }

  if (values.length === 0) {
    return null;
  }

  return sum;
}

File: responses/JQHT63-A6A/sum_of_certain_multiples.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    # write your code in Python
    ans = list(range(0,x))
    new_list = [i for i in ans if (i % 3 == 0) | (i % 5 == 0) & (i % 15 != 0)]
    return sum(new_list)
    pass